paddle/operators/proximal_gd_op.cu

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
The indentation of the license has the problem. The correct indentation is like that in `proximal_gd_op.cc`.